app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+
+
+    validates :username, presence: true,uniqueness: true,length:{minimum:3,maximum:16}
+    validates :email, presence: true,uniqueness: true
+    validates :password, presence: true,length:{minimum:3,maximum:16}
+    validates :password, :confirmation => true
```

Comment:
Could this be appended to the previous line instead of adding a new entry for it? 
Such that it becomes `validates :password, presence: true,length:{minimum:3,maximum:16}, :confirmation => true`

Reply:
Yes i probably added that validation later on and forgot to add it as one. Thank you, i will fix it now. :D